cli/dcoscli/marathon/main.py

```diff
@@ -12,6 +12,7 @@
     dcos marathon app start [--force] <app-id> [<instances>]
     dcos marathon app stop [--force] <app-id>
     dcos marathon app update [--force] <app-id> [<properties>...]
+    dcos marathon app scale [--force] <app-id> <instances>
```

Comment:
Please remove this command. We are using the save verb (`scale`) to mean two different things. The user is already able to to `dcos marathon app update <app-id> instances=<instances>`.